kube-client/src/client/config_ext.rs

```diff
@@ -220,3 +223,16 @@ impl ConfigExt for Config {
         Ok(https)
     }
 }
+
+impl Config {
+    fn exec_identity_pem(&self) -> Option<Vec<u8>> {
```

Comment:
We should document why we're doing this based on https://github.com/kube-rs/kube/issues/1088